halpybot/packages/announcer/announcer.py

```diff
@@ -209,7 +211,9 @@ class Announcement:
     edsm: Optional[int] = None
     type: Optional[str] = None
 
-    async def format(self, args: Dict, client: HalpyBOT) -> str:
+    async def format(
+        self, args: Dict[str, Union[str, int, bool]], client: HalpyBOT
```

Comment:
given that `args` has a specific structure (as you access explicit keys below), can this be a `TypedDict`?